server/storage/kv/levedb_kv.go

```diff
@@ -72,6 +78,11 @@ func (kv *LevelDBKV) Save(key, value string) error {
 	return errors.WithStack(kv.Put([]byte(key), []byte(value), nil))
 }
 
+// SaveWithTTL not supported on LevelDBKV
+func (kv *LevelDBKV) SaveWithTTL(key, value string, ttlSeconds int64) error {
```

Comment:
Hmm, it was required by generated `docs/swagger/docs.go`, but that file was not removed in `Make clean`
So if I ran `go mod tidy` after a `make build`, this will be added automatically.